rocky/rocky/views/ooi_view.py

```diff
@@ -65,10 +70,34 @@ def get_ooi_types(self) -> set[type[OOI]]:
             return self.ooi_types
         return {type_by_name(t) for t in self.filtered_ooi_types if t not in _EXCLUDED_OOI_TYPES}
 
+    def get_order_by(self) -> Literal["object_type", "scan_level"]:
+        """This ordering is limited for object type and scan level"""
+
+        return "scan_level" if self.order_by == "scan_level" else "object_type"
+
+    def get_sorting_order(self) -> Literal["asc", "desc"]:
+        return "desc" if self.sorting_order == "desc" else "asc"
```

Comment:
Since these function as properties, it would be nicer to create them as properties, e.g.:

```python3
@property
def order_by(self) -> Literal["object_type", "scan_level"]::
    return "scan_level" if self.request.GET("order_by", "") == "scan_level" else "object_type"
```
The same goes for sorting order

Reply:
Fixed in: [9b2fec5](https://github.com/minvws/nl-kat-coordination/pull/3320/commits/9b2fec5cfa8072fbb4ac785195cc7ee977cbe630)